src/web/lib/hakuneko/frontend@classic-light/jobs.html

```diff
@@ -178,6 +221,41 @@
                 }
             }
 
+            /**
+             *
+             */
+            getPlayPauseTooltip( status ) {
+                if ( status ) {
+                    return 'Resume download queue';
+                } else {
+                    return 'Pause download queue';
+                }
+            }
+
+            /**
+             *
+             */
+            getPlayPauseClass( status ) {
+                if ( status ) {
+                    return 'fa-play';
+                } else {
+                    return 'fa-pause';
+                }
+            }
+
+            /**
+             *
+             */
+            playPauseQueue() {
+                if(this.queuePaused) {
+                    Engine.DownloadManager.startQueue();
+                    this.queuePaused = false;
+                } else {
+                    Engine.DownloadManager.pauseQueue();
+                    this.queuePaused = true;
```

Comment:
The methods `DownloadManager.pauseQueue()` and `DownloadManager.startQueue()` can be called from anywhere in the software, so `this.queuePaused` may not be updated correctly.

Consider to introduce an event mechanism to update `this.queuePaused` whenever the pause state of the queue in the `DownloadManager` is changed instead of setting it manually.